iphone/Classes/TiUtils.h

```diff
@@ -155,6 +155,13 @@ typedef enum
 +(UIImage*)toImage:(id)object proxy:(TiProxy*)proxy;
 
 /**
+ Changes to image rotation, so the image is facing up.
+ @param Image to be rotated.
```

Comment:
Wrong code-doc format. Needs to have the name of the signature-property, e.g. `@param image The image to be rotated.`